http/swagger.yaml

```diff
@@ -51,3 +52,65 @@ paths:
           description: Message discarded due to invalid or missing content type.
         500:
           description: Unexpected server-side error occurred.
+definitions:
+  SenML:
+    type: object
+    properties:
+      bn:
+        type: string
+        description: Base Name
+      bt:
+        type: number
+        format: double
+        description: Base Time
+      bu:
+        type: number
+        format: double
+        description: Base Unit
+      bv:
+        type: number
+        format: double
+        description: Base Value
+      bs:
+        type: number
+        format: double
+        description: Base Sum
+      bver:
+        type: number
+        format: double
+        description: Version
+      n:
+        type: string
+        description: Name
+      u:
+        type: string
+        description: Unit  
+      v:
+        type: number
+        format: double
+        description: Value
+      vs:
+        type: string
+        description: String Value
+      vb:
+        type: boolean
+        description: Boolean Value
+      vd:
+        type: string
+        description: Data Value
+      s:
+        type: number
+        format: double
+        description: Value Sum
+      t:
+        type: number
+        format: double
+        description: Time
+      ut:
+        type: number
+        format: double
+        description: Update Time
```

Comment:
You can add here the parameters that are required. Something like 
```
    required:
      - n
      - t
```

I'm not sure how to explain that `bt`and `bn`  but I would at least mention the minimum that you need to pass the validator

Reply:
I'm in dilemma here, because the spec says "Value of the entry.  Optional if a Sum value is present; otherwise, it's required." So there is no singular field that is by itself required. Please, cf. https://tools.ietf.org/html/rfc8428

Reply:
Right, you can omit `t` but I think that `v` and `n` are always required

Reply:
Yes, hard to explain with a swagger